scripts/lint_bash.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash -v
-# shellcheck disable=SC2039
 
-pushd "$(dirname "${0}")/.." > /dev/null
+pushd "$(dirname "${0}")/.." > /dev/null || exit 1
 ROOT_DIR=$(pwd)
-popd > /dev/null
+popd > /dev/null || exit 1
```

Comment:
I'd rather call the script in a fashion that makes it fail if anything fails (`bash -eu`), rather than having it full with this nonsense. If you have to fight the linter, maybe it's a bad linter :(

Reply:
I removed `|| exit 1`, added `set -e` in the head of each file, the linter was passed. and ran the script. `lint_bash.sh` works. `integration/run.sh` does not work. I do not want to fix the issue of `integration/run.sh`, because it is outdated script.

Below is other possibilities to fix. Which do you like or any other suggestions?
SC2164 is the target lint check code for this time.

1. `scripts/lint_bash.sh`
  a. Add  `# shellcheck disable=SC2164` to the top of the file.
  b. Add `set -e` to the top of the file.
2. `tests/integration/run.sh`.
  b. Add  `# shellcheck disable=SC2164` to the top of the file.

or

3. Stop using `ShellCheck`.

Reply:
I'd rather stop using `ShellCheck`, but that's up to you.

Reply:
@hroncok I see, you would prefer the "3. Stop using ShellCheck".
@khardix how do you think?

Reply:
I changed my mind. I like to stop using `ShellCheck`. I do not want to fight for the linter continuously.
If it would be python package, I could run specific version's `ShellCheck` easily.